### PR TITLE
Olwa dev

### PR DESCRIPTION
**Pull request by owalk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,5 @@ Authors: Oliver Walker, Sean Nishi, Myles Rush, Justin Walsh
 We do not own anything! Please don't sue us.
 
 Lunk-Game is a sidescroller game inspired by the Cyanide and Happiness videos Legends of Lunk.
+
+TEST
```
Comment (owalk) [inline README.md]: this should not be all caps

Comment (owalk) [reply README.md]: this has been addressed

Comment (Sean-Nishi) [reply README.md]: Added 'test' to line 9

**Review by Sean-Nishi — APPROVED** [review]
Added 'Test 2' to line 9. Should be good for the example.

**Review by DupontDave — APPROVED** [review]

**Pull request merged**